stories/core/Buttons/ButtonGroup.stories.tsx

```diff
@@ -37,3 +54,61 @@ export const WithIcons: Story<ButtonGroupProps> = () => {
     </ButtonGroup>
   );
 };
+
+export const Overflow: Story<ButtonGroupProps> = (args) => {
+  const buttons = Array(10)
+    .fill(null)
+    .map((_, index) => (
+      <IconButton
+        key={index}
+        onClick={() => action(`Clicked on button ${index + 1}`)()}
+      >
+        <SvgPlaceholder />
+      </IconButton>
+    ));
+
+  return (
+    <div
+      style={{
+        maxWidth: 'clamp(300px, 50%, 100%)',
+        border: '1px solid hotpink',
+        padding: 8,
+      }}
+    >
+      <ButtonGroup
+        style={{ display: 'flex' }}
```

Comment:
Not really nice that it doesn't work out of the box and need to change from `inline-flex` to `flex` 😕

Reply:
Agree :O

Reply:
Oh yeah, good catch. This needs to be fixed in CSS. I put this here 2 weeks ago and forgot

Reply:
Created https://github.com/iTwin/iTwinUI/pull/247

Reply:
Changed to conditional 100%